src/main/java/zmaster587/advancedRocketry/item/ItemOreScanner.java

```diff
@@ -120,7 +119,7 @@ public void interactSatellite(SatelliteBase satellite,EntityPlayer player, World
 	@Override
 	public List<ModuleBase> getModules(int id, EntityPlayer player) {
 		List<ModuleBase> modules = new LinkedList<ModuleBase>();
-		modules.add(new ModuleOreMapper(0, 0));
+		//modules.add(new ModuleOreMapper(0, 0));
```

Comment:
???

Reply:
Wasn't needed, and it makes it more confusing. Showed it commented out to hammer that in. Satellite works without it.